src/booleans.js

```diff
@@ -1,61 +1,90 @@
 function negate(a) {
   // your code here
+  return !a;
 };
 
 function both(a, b) {
   // your code here
+  return a && b;
 };
 
 function either(a, b) {
   // your code here
+  return a || b;
 };
 
 function none(a, b) {
   // your code here
+  return !(a || b);
 };
 
 function one(a, b) {
   // your code here
+  return (a && !b) || (!a && b);
 };
 
 function truthiness(a) {
   // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
   // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
   // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
   // your code here
+  return a <= b;
 };
 
 function isOdd(a) {
   // your code here
+  return a % 2 !==0;
 };
 
 function isEven(a) {
+  return a % 2==0;
   // your code here
 };
 
 function isSquare(a) {
   // your code here
+  if (Number.isInteger(Math.sqrt(a)) == true) {
+    return true;
+  } else {
+    return false;
+  };
 };
 
 function startsWith(char, string) {
   // your code here
+  return string.startsWith(char);
 };
 
 function containsVowels(string) {
   // your code here
+  string = string.toLowerCase();
```

Comment:
The name string is already being used as the passed in parameter if the containsVowels function. 
If you would like to assign 
```
string.toLowerCase()
```
 to a variable you will need to declare a new variable with a new suitable name, for example:
`````
const lowerCaseString = string.toLowerCase();
`````
and then use the new string in your for loop and in the if statement.